File: AikoCRUDAPI/AikoCRUDAPI/Models/EquipmentPositionHistory.cs
namespace AikoCRUDAPI.Models
{
    public class EquipmentPositionHistory
    {
        public Guid equipment_id { get; set; }
        public DateTime date { get; set; }
        public double lat { get; set; }
        public double lon { get; set; }
    }
}

File: src/main/java/com/equipments/apirest/models/EquipmentPositionHistory.java
package com.equipments.apirest.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.EmbeddedId;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.time.LocalDate;

@Entity
@NoArgsConstructor
@Getter
@Setter
@Table(name = "equipment_position_history", schema = "operation")
public class EquipmentPositionHistory {

    @EmbeddedId
    private EquipmentPositionHistoryId equipmentPositionHistoryId;

    private LocalDate date;

    private Float lat;

    private Float lon;

}
